caffe2/utils/math/elementwise_level2.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CAFFE2_UTILS_MATH_ELEMENTWISE_H_
+#define CAFFE2_UTILS_MATH_ELEMENTWISE_H_
+
```

Comment:
Can we have a more meaningful name for this header/cc file? And add some comments talking about which functions should be put in this file?

Reply:
The AffineChannel is a special case of elementwise mul and add operation, so I think it's fine to make it in elementwise.h